api-specs/api/api.raml

```diff
@@ -137,6 +137,19 @@ resourceTypes:
         body:
           example: !include examples/project.example.json
           type: Project
+  head:
+    displayName: Get project settings with head
+    is:
+      - errorable
+      - deprecatable200
+    description: |
+      The Endpoint is responding a limited set of information about settings and configuration of the project.
+    securedBy: [oauth_2_0: { scopes: ['manage_project:{projectKey}'] }]
+    responses:
+      200:
+        body:
+          example: !include examples/project.example.json
+          type: Project
```

Comment:
FYI: `HEAD` requests do not have a response body usually.

Not sure if we can omit this. cc @nkuehn

Reply:
I would keep it as a smoke test because although it's not just "usually" but clearly forbidden in the the HTTP spec of HEAD nothing prevents a RAML author from actually defining a body in the RAML / openAPI.